docs/src/meshes/p4est_mesh.md

```diff
@@ -12,3 +12,224 @@ such as the compressible Euler equations can use [`FluxRotated`](@ref) to
 wrap numerical fluxes implemented only for Cartesian meshes. This simplifies
 the re-use of existing functionality for the [`TreeMesh`](@ref) but is usually
 less efficient, cf. [PR #550](https://github.com/trixi-framework/Trixi.jl/pull/550).
+
+## Construction of P4estMesh from an Abaqus file
+
+One available option to construct a [`P4estMesh`](@ref) is to read in an Abaqus (`.inp`) mesh file.
+We briefly describe the structure of this file, the conventions it uses, and how the mesh file
+is parsed to create an initial unstructured, curvilinear, and conforming mesh.
+
+For this discussion we use the following two-dimensional unstructured curved mesh with three elements:
+
+![abaqus-mesh-docs](https://user-images.githubusercontent.com/25242486/139241997-88e70a01-286f-4cee-80b1-2fd83c60bcca.png)
+
+We note that the node and element connectivity information parsed from the Abaqus file creates
+a straight sided (linear) mesh.
+From this linear mesh there are two strategies available to make the mesh curvilinear:
+
+1. Apply a `mapping` function to describe a transformation of the linear mesh to another
+   physical domain. The mapping is approximated using interpolation polynomial of a user
+   specified polynomial degree. The default value of this polynomial degree is `1` that
+   corresponds to an uncurved geometry.
+2. High-order boundary information is available in the `.inp` mesh file because it was created
+   with the [`HOHQMesh`](https://github.com/trixi-framework/HOHQMesh.jl) generator.
+   This information is used to create appropriate transfinite mappings during the mesh construction.
+
+We divide our discussion into two parts. The first part discusses the standard node and element information
+contained in the `.inp` mesh file. The second part specifically deals with the mesh file parsing of an Abaqus
+file created by `HOHQMesh`.
```

Comment:
```suggestion
file created by HOHQMesh.jl.
```
As stated above, I would change this (and future) occurrences to HOHQMesh.jl. Users not from Julia will not read this or not care either way, and Julia users are (hopefully) less confused about whether they need to use HM or HM.jl

Reply:
I don't have a strong opinion on this. It just seems weird to write into the Fortran output routine that is was generated with a Julia package ;)

Reply:
No, this is not the output of the Fortran routine, this is just the last part of the sentence
> The second part specifically deals with the mesh file parsing of an Abaqus *file created by `HOHQMesh.jl`.*

(italics are mine for emphasis)

Or am I the confused one here?

Reply:
Ah sorry, I did get confused. I thought you meant to also change the file header that we use to check the mesh file read-in strategy.

Reply:
No, no, this line from HOHQMesh I wouldn't change, you are absolutely right! I am just referring to our docs here.